### ISSUE TITLE
Support ColdFusion comments

### ISSUE THREAD

**Issue by bebraw** [MonoLisaFont/feedback#86]
Code:

```
 ColdFusion comments needs improvements
    <!--- <cfset foo = "Hello World!"> --->
 
    HTML Comment Looks Great!
    <!-- <h1>My Site</h1> -->
```

How it looks now:

![image001](https://user-images.githubusercontent.com/166921/120694236-9fd9b580-c4b2-11eb-8c8f-5a99100b6b7c.png)

**Comment by okonet** [comment]
We agreed to add 2 new ligatures to accommodate this use case.

**Comment by MarcusSterz** [comment]
Added. Will be available in 1.802.

**Comment by bebraw** [comment]
Implemented in 1.802.
